clientv3/client.go

```diff
@@ -490,6 +450,21 @@ func newClient(cfg *Config) (*Client, error) {
 	client.Auth = NewAuth(client)
 	client.Maintenance = NewMaintenance(client)
 
+	{
```

Comment:
Do we have a special reason to make this part as a block? If we have it's fine of course.

Reply:
yeah, just want to keep the variables clean.  I'll remove it.